.travis.yml

```diff
@@ -77,10 +80,16 @@ matrix:
            CONDA_DEPENDENCIES=${CONDA_ALL_DEPENDENCIES}
            EVENT_TYPE='cron'
 
+before_install:
+    - sudo apt-get install python-virtualenv
+
 install:
   - git clone git://github.com/astropy/ci-helpers.git
-  - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+  # - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
   # additional external tools (Issue #898) -- HOLE
+  - virtualenv venv --system-site-packages
```

Comment:
Also dependencies like netcdf will be harder to install. You will likely need to add some apt packages.